Gir.toml

```diff
@@ -0,0 +1,35 @@
+[options]
+girs_dir = "gir-files"
+library = "GLib"
+version = "2.0"
+min_cfg_version = "2.32"
+target_path = "."
+work_mode = "normal"
+
+generate = [
+    "GLib.KeyFileFlags",
+]
+
+ignore = [
+]
+
+manual = [
+    "GLib.Bytes",
+    "GLib.Error",
+    "GLib.Variant",
+    "GLib.VariantType",
+    "GObject.Object",
+]
+
+[[object]]
+name = "GLib.KeyFile"
+status = "generate"
+       [[object.function]]
+       name = "load_from_data_dirs"
+       ignore = true
+       [[object.function]]
+       name = "load_from_dirs"
+       ignore = true
+       [[object.function]]
+       name = "error_quark"
+       ignore = true
```

Comment:
Since the error enum is generated correctly never mind, this `ignore` seems the least hairy solution.